stories/ChartKit/index.js

```diff
@@ -1,2 +1,5 @@
+// @TODO File import issues Commented ou for now
 //export { default } from './HorizontalBar';
-export { default } from './ProgressBar';
+
+// @TODO  File import issues due to analytics code bundled into the ProgressBar component
+//export { default } from './ProgressBar';
```

Comment:
we can put this back now :) just drop https://github.com/kids-first/kf-portal-ui/pull/1129/commits/8d5eccb205c2f3f3f2f95f49d0acaa00781f60f7 or revert it 📦

Reply:
merged to latest